src/Core/src/Platform/iOS/ContentView.cs

```diff
@@ -37,7 +47,90 @@ public override void SetNeedsLayout()
 			Superview?.SetNeedsLayout();
 		}
 
+		public new void Dispose()
+		{
+			DisposeClip();
+
+			base.Dispose();
+		}
```

Comment:
Is the UIView not disposable? Should we override?

Also, @PureWeen said things about never disposing, so I wonder what this means. Should we let the GC take control?

Reply:
@jsuarezruiz  do you need this to fix the issue?